src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -55,7 +55,7 @@ export type DetailsViewCommandBarDeps = {
     reportGenerator: ReportGenerator;
     getDateFromTimestamp: (timestamp: string) => Date;
     detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
-    assessmentActionMessageCreator: AssessmentActionMessageCreator;
+    getAssessmentActionMessageCreator: () => AssessmentActionMessageCreator;
```

Comment:
nit: It seems like we're not using this here; we should probably remove it entirely.